src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -398,6 +366,97 @@ public void ContinueAsNew(object input)
             this.ContinuedAsNew = true;
         }
 
+        internal async Task<TResult> CallDurableTaskFunctionAsync<TResult>(string functionName,
```

Comment:
Can this method be private? If so, let's make it private. It's good to minimize visibility for both better encapsulation but (IMO more importantly) for better code understandability/maintainability.